SAMLBridgeConfiguation/SAMLBridgeConfiguation.csproj

```diff
@@ -0,0 +1,135 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
Do these 4.0 references scattered around refer to the .NET Framework version, and do they mean that this tool (and possibly also SAML Bridge) requires version 4.0 to run? In some places here, the old values from GSARKS were 2.0 or 3.5.